### DISCUSSION TITLE
STOx'sのデフェンスの強さを研究する

### DISCUSSION THREAD

**Discussion by HansRobo** [ibis-ssl/crane#272] [General]
https://youtu.be/YIWJbFjSOe8?si=0LutUU0lITUycuux&t=340

**Comment by HansRobo** [comment]
基本陣形
![image](https://github.com/ibis-ssl/crane/assets/12395284/d36a8188-e39b-466d-aafb-c57ed75f8eed)

**Comment by HansRobo** [comment]
前進守備を多用している
![image](https://github.com/ibis-ssl/crane/assets/12395284/643dde50-fda8-4d48-948d-eb407dab99f0)

![image](https://github.com/ibis-ssl/crane/assets/12395284/3fe67683-7ea1-4d1d-a767-afe79308593f)

**Comment by HansRobo** [comment]
逆に点を入れられているのは前進守備ができていない時

![image](https://github.com/ibis-ssl/crane/assets/12395284/3a8df99c-6e6c-4c56-ab2e-301b6a638270)

**Comment by HansRobo** [comment]
2017年のTDPの3章「Defensive Strategies」より

## マーク

基本の防衛戦略はマンツーマンのマークである

特筆事項

- マーク距離は敵とゴールに比例し、近いほど短くしている
- マーカーの移動距離は最短となるようにターゲットを割り振られる

**Comment by ssaattww** [comment]
# ibisでの、対応方針案
ゴーリーとディフェンダーの座標生成を、total_defense_plannerで行う。

## 変更するもの
- goalie ボールがペナルティエリア外時 ロボット座標指示
- defender 壁の作る座標(1台目)
  - 壁1台目の座標はgoalieにして、開ける 
## 流用するもの
- goalie ボール排出等、ペナルティエリア内の動作
- goalie パス先の処理
- defender 壁の作る座標(2台目以降)
